monero-outreach-docs/translations/de/merchants/accept_monero_stickers_de.md

```diff
@@ -0,0 +1,14 @@
+# Wir akzeptieren Monero Aufkleber
+
+Monero Outreach bevorzugt die Formulierung _Wir akzeptieren Monero_, anstatt dem Beispiel von Bitcoin mit _Monero wird hier akzeptiert_ zu folgen. Bei _Wir_ geht es um Menschen, es geht darum, dass Menschen auf eine neue Art und Weise über Geld nachdenken. Es ist nur eine subtile Abweichung der Formulierung, aber wir denken, es sagt viel über die Absicht aus. Wir bitten Dich um Hilfe bei der Verbreitung dieser Absicht.
```

Comment:
I'll use "Es ist nur eine subtile Verschiebung der Phrasierung" instead of "Es ist nur eine subtile Abweichung der Formulierung"

Reply:
I am not familiar with the word "Phrasierung" and duden locates it in music, but "Abweichung" I think is better at this point.

Reply:
Okay. It was a suggestion so if you prefer that it's ok.